trpcultivate_phenotypes/src/Plugin/Validators/DuplicateTraits.php

```diff
@@ -27,60 +22,70 @@
  */
 class DuplicateTraits extends TripalCultivatePhenotypesValidatorBase implements ContainerFactoryPluginInterface {
 
-  /**
-   *   This validator requires the following validator traits:
-   *   - GenusConfigured: Gets a string of the configured genus name
-   *   - ColumnIndices => Gets an associative array with the following keys,
-   *       which are column headers of required columns for the Traits Importer:
-   *     - 'Trait Name': int, the index of the trait name column in $row_values
-   *     - 'Method Short Name': int, the index of the method name column in $row_values
-   *     - 'Unit': int, the index of the unit column in $row_values
+  /*
```

Comment:
```suggestion
  /**
```

Reply:
Done in commit [9c6f567](https://github.com/TripalCultivate/TripalCultivate-Phenotypes/pull/119/commits/9c6f567d684d387d84a9610e068af53466e3d95f)!